test/fail_compilation/diag14875.d

```diff
@@ -67,7 +64,7 @@ deprecated template Vaz(T)
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag14875.d(75): Error: static assert:  `0` is false
+fail_compilation/diag14875.d(72): Error: static assert:  `0` is false
```

Comment:
Just remove this and use `-de` ?

Reply:
Can do but I usually try to keep changes to existing tests as small as possible